libs/sdk-core/src/time_utils.rs

```diff
@@ -0,0 +1,26 @@
+use std::time::{SystemTime, UNIX_EPOCH};
+
+pub(crate) struct BreezTimeUtils;
```

Comment:
I think making this a trait only to be able to test created_at is overkill and we can test that without the trait.
IMO it is fine to get the current time directly from rust API.